### ISSUE TITLE
opennebula-sunstone Ubuntu package should not provide an empty /var/lib/one/sunstone/main.js

### ISSUE THREAD

**Issue by baby-gnu** [OpenNebula/packages#54]
* opennebula-sunstone ubuntu 18.04 package version 5.6.1-1

The `debsums` utility checks the checksum of installed files and report any modification.

When using it with a `debsums -c` I found that `/var/lib/one/sunstone/main.js` was modified.

The package `opennebula-sunstone` install an empty file but now my file is 6.5MB:

```
# ls -lh /var/lib/one/sunstone/main.js /root/one-sunstone-pkg/var/lib/one/sunstone/main.js
-rw-r--r-- 1 root     root        0 sept. 20 13:41 /root/one-sunstone-pkg/var/lib/one/sunstone/main.js
-rw-r--r-- 1 oneadmin oneadmin 6,5M janv.  3 13:32 /var/lib/one/sunstone/main.js
```

Their respective SHA256

```
# sha256sum /var/lib/one/sunstone/main.js /root/one-sunstone-pkg/var/lib/one/sunstone/main.js
e5c6f60b8daa0835642f0d1c42070a7b9452ebcf5b2d5cdf7e3c76f6fb2c80f3  /var/lib/one/sunstone/main.js
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855 /root/one-sunstone-pkg/var/lib/one/sunstone/main.js
```

If the purpose is to have a non empty `/var/lib/one/sunstone/` directory, the packaging should provide a [debian/opennebula-sunstone.dirs](https://manpages.debian.org/testing/debhelper/dh_installdirs.1.en.html).

Regards.

**Comment by vholer** [comment]
Fixed by 5612253b2ace41ad70ac2e9efd9ed3ae3ff4a1aa.

`/var/lib/one/sunstone/main.js` is now created / removed by the post-un/install scripts.

Verification doesn't complain:

```
# debsums -c opennebula-sunstone && echo OK
OK

# rpm -V opennebula-sunstone && echo OK
OK
```

Thanks for the report.

**Comment by baby-gnu** [comment]
Thanks a lot
